src/Cli/Utils.cs

```diff
@@ -904,27 +904,37 @@ public static bool WriteJsonContentToFile(string file, string jsonContent)
         /// This method will check if DAB_ENVIRONMENT value is set.
         /// If yes, it will try to merge dab-config.json with dab-config.{DAB_ENVIRONMENT}.json
         /// </summary>
-        public static string MergeConfigsIfAvailable()
+        /// <returns>Returns the name of the merged Config if successful.</returns>
```

Comment:
can you also note that this would return the file name only (not full/relative path) in the following format?
- `{CONFIGFILE_NAME}.{environmentValue}.merged{CONFIG_EXTENSION}`

Reply:
yep, like how we deal with other config naming through out